modules/object-collections.adoc

```diff
@@ -0,0 +1,29 @@
+// Module included in the following assemblies:
+//
+// * operating/create-use-collections.adoc
+:_content-type: CONCEPT
+[id="understanding-object-collections_{context}"]
+= Understanding object collections
+
+You can create an object collection that can be used across namespaces, clusters, and deployments. Some common use cases for collections include:
+
+* Creating a group of namespaces to identify an infrastructure area that is owned by a specific development team
+* Using a deployment label to group all of the deployments in a distributed application spanning multiple namespaces
+* Identifying an application used in used in multiple clusters, such as a development and production cluster, which requires the same policy regardless of location
+* Performing cluster mapping for an entire production or test environment
+* Using annotations to identify the owner of a vulnerability issue that impacts a range of deployments or images
+
+See the "Collections configuration examples" section for specific use case examples.
+
+You can define collections using both _explicit identifiers_, such as Cluster ID, Namespace ID, and Deployment ID and _implicit identifiers_, such as Kubernetes labels and annotations. Collections are dynamic, or resolved in runtime. Collections can be hierarchical and include other collections.
```

Comment:
```suggestion
You can define collections using both _explicit identifiers_, such as Cluster Name, Namespace Name, and Deployment Name and _implicit identifiers_, such as Kubernetes labels. Collections are dynamic, or resolved in runtime. Collections can be hierarchical and include other collections.
```
We don't have a way to build collections from the object IDs that ACS assigns to Clusters/NS/Deployments, they must be selected by name. I'm not sure if customers use the term ID interchangeably with name but I think it is worth making the distinction. e.g. It is very possible that a user creating a collection with a single rule that targets a namespace with name `stackrox` will include multiple namespaces with that name across different clusters. 

The current UI doesn't support a way to select _exactly one instance_ of the stackrox namespace, unless a specific cluster name is paired with the rule as well. A collection with "Cluster Name=production" and "Namespace Name=stackrox" would be equivalent to selecting a namespace by ID.

Also removed `annotations` based on my previous comment above.

Reply:
Boaz rewrote this section, but I've noted for other parts of the doc.